compiler/ETA/CodeGen/Types.hs

```diff
@@ -157,7 +158,7 @@ mkCgIdInfoWithLoc id lfInfo cgLoc =
            , cgLocation = cgLoc }
 
 mkStaticLoc :: DynFlags -> Id -> LambdaFormInfo -> CgLoc
-mkStaticLoc dflags id _ = LocStatic closureType modClass clName
+mkStaticLoc dflags id _ = LocStatic (obj (qualifiedName modClass clName)) modClass clName
```

Comment:
`locFt` used throughout the codegen will extract this field and some corner cases may break the verifier because of this. Can you explain why you made this change? I don't see you using this change anywhere.